### DISCUSSION TITLE
Zoom inside object

### DISCUSSION THREAD

**Discussion by jluism** [google/model-viewer#2121] [Q&A]
Good morning

I have created terrain model with some buildings using a drone and now im trying to show it in web using model-viewer. You can see it in https://www.asturnatura.com/3dprueba4.php. However, i would like more zoom into buildings, but i think that model viewer does not zoom inside object limit. In obj object or in dronedeploy (the tool used to generate the model) is possible to zoom more and also see windows with more details.

Do you have any idea about how to increase the zoom detail?

Thanks

**Comment by jluism** [comment]
This is the suggested zoom i would like to have, attached, but zoom in model-viewer does not allow to me

![Untitled-1](https://user-images.githubusercontent.com/27965085/110829746-c237aa80-8298-11eb-8b2c-4c9614e0136c.jpg)

**Reply by elalish** [reply]
Yes, simply set https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-minFieldOfView to a small number of degrees; this will zoom in like a telescope. If you'd rather, you can also set a smaller minimum radius to get physically closer: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-minCameraOrbit, but the problem is then you might enter the building and start seeing the backside of the mesh. You may also want to enable more general interactions like panning, which there's a JS example of here: https://modelviewer.dev/examples/stagingandcameras/#panning

**Reply by jluism** [reply]
Thank you elalish, this is very helpful
